src/calendar/MonthView.java

```diff
@@ -0,0 +1,241 @@
+package calendar;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import javax.swing.*;
+
+public class MonthView extends JPanel {
+	// Private instance variable used to navigate between months
+	private int n = 0;
+
+	/*
+	 * Constructor which updates the value of n and calls the display method to
+	 * display the calendar.
+	 * 
+	 * @param n is the month you want to show (either before or after the current
+	 * month)
+	 */
+	public MonthView(int n) {
```

Comment:
constructor should take in a CalendarModel as a parameter and store a reference to it in the class